sdk/formrecognizer/Azure.AI.FormRecognizer/src/RecognizeContentOperation.cs

```diff
@@ -102,9 +102,6 @@ private async ValueTask<Response> UpdateStatusAsync(bool async, CancellationToke
                     ? await _serviceClient.GetAnalyzeLayoutResultAsync(new Guid(Id), cancellationToken).ConfigureAwait(false)
                     : _serviceClient.GetAnalyzeLayoutResult(new Guid(Id), cancellationToken);
 
-                // TODO: Handle correctly according to returned status code
-                // https://github.com/Azure/azure-sdk-for-net/issues/10386
-
```

Comment:
Would it make sense to handle a Failed OperationStatus the same way for receipts and content as we're doing in custom form?  If the operation failed for some reason, we might similarly throw the wrong exception type here by trying to deserialize the PageResults & ReadResults from the response body.  

If we don't know of an instance of this happening in the service currently, it would be good to file an issue to track it for Preview 3.

Reply:
I think that filing an issue is a good idea. Like you suggested, I'm not aware of a scenario where this could be an issue. Receipt and content recognition seem to be way more resilient to these kind of errors than the custom endpoint.

Reply:
#11474